main.css

```diff
@@ -0,0 +1,150 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  position: relative;
+  height: 85vh;
+  border: 1px solid #333;
+  background-image: url("./images/apple_background.jpg");
+  background-position: 50% 100%;
+}
+
+nav {
+  position: relative;
+  top: 15%;
+  left: 17%;
+  background: #333 linear-gradient(180deg, #999, #333);
+  border: 0.1px solid #333;
+  border-radius: 5px;
+  width: 65%;
+  height: 7%;
+}
```

Comment:
The requirements in this project is to use flex, grid or float, I believe you can achieve a good result with this element in this example, please change the solution from using position: relative to flex, grid or float.
- [ ] Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) and  [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/))

Reply:
If you need position: relative to make position: absolute work in a child element, you can keep it, but the positioning of the element should be with flex, grid or float.